src/main/java/games/loveletter/actions/deep/DeepBaronAction.java

```diff
@@ -0,0 +1,36 @@
+package games.loveletter.actions.deep;
+
+import core.AbstractGameState;
+import core.actions.AbstractAction;
+import games.loveletter.LoveLetterGameState;
+import games.loveletter.cards.LoveLetterCard;
+
+import java.util.List;
+
+/**
```

Comment:
General comment that these Actions have no toString() method implemented - which is very helpful in debugging, and also in metrics output and on the GUI history panel...

Reply:
They do on some super class, and I preferred this to more granular toStrings that end up confusing information more. In my tests they showed up properly in metrics / GUI